### ISSUE TITLE
error loading gff: NULL Phase?

### ISSUE THREAD

**Issue by srobb1** [tripal/tripal#271]
### System information

* Tripal Version: Tripalv2

### Issue description
I have loaded so many GFFs, and have never seen this error before. I am getting an error about a NULL phase. I have a phase. Does anyone see what is wrong with my GFF? I checked the tabs, they are okay.

These are the only lines I am loading:
```
Pca_1   maker   gene    6676    8508    .       +       .       ID=Pca_1_0.4:gene;gene_id=Pca_1_0.4:gene;Name=Putative Brain super conserved receptor;Dbxref=Gene3D:G3DSA:1.20.1070.10,InterPro:IPR000276,InterPro:IPR017452,PRINTS:PR00237,Pfam:PF00001,ProSitePatterns:PS00237,ProSiteProfiles:PS50262,SUPERFAMILY:SSF81321;Ontology_term=GO:0004930,GO:0007186,GO:0016021;Note=Similar to gpr173: Probable G-protein coupled receptor 173 (Danio rerio);ahrd_human_readable_description=Brain super conserved receptor
Pca_1   maker   mRNA    6676    8508    .       +       .       ID=Pca_1_0.4;transcript_id=Pca_1_0.4;Parent=Pca_1_0.4:gene;Name=Putative Brain super conserved receptor;_AED=0.36;_eAED=0.38;_QI=0|-1|0|1|-1|1|1|0|610;Dbxref=Gene3D:G3DSA:1.20.1070.10,InterPro:IPR000276,InterPro:IPR017452,PRINTS:PR00237,Pfam:PF00001,ProSitePatterns:PS00237,ProSiteProfiles:PS50262,SUPERFAMILY:SSF81321;Ontology_term=GO:0004930,GO:0007186,GO:0016021;Note=Similar to gpr173: Probable G-protein coupled receptor 173 (Danio rerio);ahrd_human_readable_description=Brain super conserved receptor
Pca_1   maker   exon    6676    8508    .       +       .       ID=Pca_1_0.4:1;Parent=Pca_1_0.4;transcript_id=Pca_1_0.4;gene_id=Pca_1_0.4:gene;
Pca_1   maker   CDS     6676    8508    .       +       0       ID=Pca_1_0.4:cds;Parent=Pca_1_0.4
```

Here is my error:
```
Calling: tripal_feature_load_gff3(/var/other_data/organisms/snail/Pcan/genomes/test.gff, 23, 81, 0, 1, 0, 0, 0, , , 0, , , , 0, , , 957)
Opening /var/other_data/organisms/snail/Pcan/genomes/test.gff
Parsing Line 4 (100.00%). Memory: 39,715,336 bytes.
FAILED
WD tripal_feature: PDOException: SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in      [error]
column "phase" violates not-null constraint
DETAIL:  Failing row contains (9125287, 9125285, null, 1, 6675, 8508).: INSERT INTO
chado.tripal_gffcds_temp (feature_id, parent_id, fmin, fmax, strand) VALUES (:feature_id,
:parent_id, :fmin, :fmax, :strand); Array
(
    [:feature_id] => 9125287
    [:parent_id] => 9125285
    [:fmin] => 6675
    [:fmax] => 8508
    [:strand] => 1
)
 in chado_query() (line 1538 of
/var/www/html/sites/all/modules/tripal/tripal_core/api/tripal_core.chado_query.api.inc).
```

**Comment by srobb1** [comment]
What the heck!! I changed the 0 phase to 1 and now it loads. 0 are valid phases. What is going on here??? And I KNOW I have loaded 0 phases before. I am so confused

**Comment by srobb1** [comment]
this fixed it. I am not sure why I didnt have this in my code base already, oh, well it is working now:

https://github.com/tripal/tripal/commit/0a71bba69f3c9b7ae67d2f570718e61cd1361830

**Comment by spficklin** [comment]
Just curious... What was the fix?

**Comment by srobb1** [comment]
@spficklin I found the fix here [0a71bba](https://github.com/tripal/tripal/commit/0a71bba69f3c9b7ae67d2f570718e61cd1361830)

I must have updated my tripal with drush at some point and got out of sync with github. I KNOW that I loaded 0 phase features. I just updated from github and all is good :)
